docs/platform_developers/developer_getting_started.rst

```diff
@@ -0,0 +1,151 @@
+********************************
+Developer Getting Started Guide
+********************************
+
+This page describes how to setup your development environment to:
+
+* Setup Inmanta pip artifacts
```

Comment:
The first two steps are only relevant for employees. Perhaps this has to be split or moved to the internal docs entirely. 

@bartv wdyt?

Reply:
agreed

Reply:
(bart means: yes, split)

Reply:
yes. I understood this part. 
I assumed this was to be used internally. As I see now, Gitlab is our private/internal stash? If so, could you please let me know where this doc exactly has to land?

Reply:
so the internal part should go into here: https://code.inmanta.com/inmanta/developerdocs (and refer to the public docs for the common part)